recipes/aws-noaa-whoi/recipe.py

```diff
@@ -0,0 +1,20 @@
+import s3fs
+
+from pangeo_forge_recipes.patterns import pattern_from_file_sequence
+from pangeo_forge_recipes.recipes.reference_hdf_zarr import HDFReferenceRecipe
+
+url_base = 's3://noaa-cdr-sea-surface-temp-whoi-pds/data/'
+
+years = range(1988, 2022)
+file_list = []
+
+fs = s3fs.S3FileSystem(anon=True)
+
+for year in years:
+    file_list += sorted(
+        filter(lambda x: x.endswith('.nc'), fs.ls(url_base + str(year), detail=False))
+    )
+
+pattern = pattern_from_file_sequence(file_list, 'time', nitems_per_file=1)
```

Comment:
My guess is that the files in `file_list` do not start with `s3://`, so that fsspec is looking for local files instead.

Reply:
I'm working on a separate recipe and found this is the issue. adding a map to add the missing `s3://` solve dit.

```
from os.path import join

fs = s3fs.S3FileSystem(anon=True)
is_nc = lambda x: x.endswith('.nc')
add_s3 = lambda x: "s3://" + x
for year in years:
    file_list += sorted(
        filter(is_nc, map(add_s3, fs.ls(join(url_base, str(year)), detail=False)))
    )
```

Reply:
thanks @cisaacstern @rabernat and @rbavery for catching this, I see the error here and will revise with the suggestion above. 👍